masterlist.yaml

```diff
@@ -8589,12 +8589,21 @@ plugins:
 
   - name: 'Arcanum.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/23488/' ]
-    msg: [ *patch3rdPartySRPC ]
+    msg:
+      - *patch3rdPartySRPC
+      - <<: *alsoUseX
+        subs: [ '[Arcanum - A New Age of Magic 4.0.1 Bugfix](https://www.nexusmods.com/skyrimspecialedition/mods/46322)' ]
+        condition: 'checksum("Arcanum.esp", AA67A039)'
+      - type: warn
+        content: 
+          - lang: en
+            text: 'The BSA files included with this version of Arcanum need to be renamed in order for the mod to work. Remove "4.0 BSA" from the name of each .bsa file so you are left with Arcanum.bsa and Arcanum - Textures.bsa.'
+        condition: 'checksum("Arcanum.esp", AA67A039)'
```

Comment:
```yaml
        condition: 'checksum("Arcanum.esp", AA67A039)'
```
The issue with using `Arcanum.esp` in the condition is it will always show the message unless they modify that file.
It should really be checking if the wrong BSA file is installed, and also make sure a valid version of the BSA isn't installed.
```yaml
        condition: 'not file("Arcanum.bsa") and file("Arcanum 4.0 BSA.bsa")'
```

Reply:
Oh nice! I thought you could only use plugin names, not bsas. Thank you!

Reply:
> 
> 
> Oh nice! I thought you could only use plugin names, not bsas. Thank you!

You can read more about what conditions you can use [here](https://loot-api.readthedocs.io/en/stable/metadata/conditions.html).